test/models/profile_feature_test.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/account/models/profile_feature.dart';
+
+import '../util/factories/profile_factory.dart';
+import '../util/factories/profile_feature_factory.dart';
+
+void main() {
+  group('ProfileFeature.fromJson', (() {
+    test('parses ProfileFeature from json', (() {
+      Map<String, dynamic> json = {
+        "id": 1,
+        "created_at": "2021-01-01T00:00:00.000Z",
+        "profile_id": 1,
+        "feature": Feature.noSmoking.index,
+        "rank": 1,
+      };
+      ProfileFeature profileFeature = ProfileFeature.fromJson(json);
+      expect(profileFeature.id, 1);
+      expect(profileFeature.createdAt, DateTime.parse("2021-01-01T00:00:00.000Z"));
+      expect(profileFeature.profileId, 1);
+      expect(profileFeature.feature, Feature.noSmoking);
+    }));
+
+    test('can handle Profile', (() {
+      Profile profile = ProfileFactory().generateFake();
+      Map<String, dynamic> json = {
+        "id": 1,
+        "created_at": "2021-01-01T00:00:00.000Z",
+        "profile_id": profile.id,
+        "feature": Feature.accessible.index,
+        "rank": 1,
+        "profile": profile.toJsonForApi(),
+      };
+      ProfileFeature profileFeature = ProfileFeature.fromJson(json);
+      expect(profileFeature.profile.toString(), profile.toString());
+    }));
+
+    //? Test that checks if error is thrown when feature is not in enum
```

Comment:
Everything looks good, I will approve after you add those tests :) 
(and can you add a similar one for profile as well please, we forgot it there)